### DISCUSSION TITLE
Segmentation UNET results

### DISCUSSION THREAD

**Discussion by Meddebma** [Project-MONAI/MONAI#2670] [General]
Hi, 
I was implementing the UNET segmentation tutorial and the training went well, but when I tried to display the segmentation results, the output was completely empty, is it a problem of training or a matter of display?
here is the code
https://github.com/Meddebma/pyradiomics/blob/master/Untitled0.ipynb

thank you

**Comment by Nic-Ma** [comment]
Hi @yiheng-wang-nv ,

Could you please help check the notebook?

Thanks in advance.

**Reply by Nic-Ma** [reply]
Hi @Meddebma ,

I think this ticket is duplicated, can we close it and discuss in https://github.com/Project-MONAI/MONAI/discussions/2668 directly？

Thanks.

**Reply by Meddebma** [reply]
Hi @Nic-Ma, 
as you like, the difference is that in this notebook I used the UNET tutorial with only one output_channel, in the other Ticket I am using the two channels output in the model architecture (Which works well) and I am trying to remove the background channel
